wordpress-mobile/WordPress-Android
# Issue/update color scheme

### Fix
Update the colors used in the bar chart on the ***Stats*** screen.  See the screenshots below for illustration.

![1564426900](https://user-images.githubusercontent.com/3827611/62087825-5a962e00-b220-11e9-8ef7-519890b63a4d.png)

### Test
1. Go to ***Sites*** tab.
2. Tap ***Stats*** item.
3. Go to ***Days***, ***Weeks***, ***Months***, or ***Years*** tab.
4. Notice updated colors in bar chart as shown above.

### Review
Only one developer is required to review these changes, but anyone can perform the review.  The code review should not worry about the color names or specific changes to the color of the interface, but it should check that there are no breaking changes.  The design and interface will be reviewed by @mattmiklic.
